conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureExceptions.java

```diff
@@ -72,6 +75,10 @@ public void handle(HttpServerExchange exchange, Throwable throwable) {
             illegalArgumentException(exchange, throwable);
         } else if (throwable instanceof FrameworkException) {
             frameworkException(exchange, (FrameworkException) throwable);
+        } else if (throwable instanceof DeadlineExpiredException.External) {
+            externalDeadlineExpiredException(exchange, (DeadlineExpiredException.External) throwable);
+        } else if (throwable instanceof DeadlineExpiredException.Internal) {
+            internalDeadlineExpiredException(exchange, (DeadlineExpiredException.Internal) throwable);
```

Comment:
We might as well handle this in one branch, that way new types of `DeadlineExpiredException` are handled without code change (caveat being that the logging may not be ideal for that case, but behavior should be reasonable)

Reply:
done